build.gradle

```diff
@@ -41,8 +41,10 @@ allprojects {
     compile 'net.savantly:graphite-client:1.1.0-RELEASE'
     compile 'com.timgroup:java-statsd-client:3.0.1'
     compile 'com.signalfx.public:signalfx-codahale:0.0.47'
+    compile 'org.mockito:mockito-core:2.24.0'
```

Comment:
Why does this need a compile time dependency on mockito?  testCompile should be sufficient.